Automattic/wp-calypso
# Site Profiler: Add collapsed get report form when submitting

Resolves https://github.com/Automattic/dotcom-forge/issues/7457
Depends on https://github.com/Automattic/dotcom-forge/issues/7308

## Proposed Changes

* Add a collapsed version with a success message when submitting the Get Report Form

|Collapsed form |
|-------|
| ![CleanShot 2024-05-28 at 16 45 47@2x](https://github.com/Automattic/wp-calypso/assets/3519124/a4092fac-094f-4ce2-8439-590ae220e3d4)|


## Why are these changes being made?


* To display a Success notification to the user as per Figma

## Testing Instructions



* Navigate to the Site Profiler and add a site, e.g. `/site-profiler/example.com`
* Click on the `Access full site report - It’s free` main CTA button
* On the form that slides up from the bottom, populate correctly all the details and Submit the form
* Make sure the styles match Figma design and the responsive behaviour is fine

## Pre-merge Checklist



~~- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?~~
